code/game/objects/structures/displaycase.dm

```diff
@@ -319,7 +319,8 @@ var/global/list/captain_display_cases = list()
 				else
 					to_chat(src, "[bicon(src)] <span class='warning'>\The [src] is empty!</span>")
 		else
-			user.visible_message("[user.name] gently runs his hands over \the [src] in appreciation of its contents.", \
+			user.changeNext_move(CLICK_CD_MELEE)
+			user.visible_message("[user.name] gently runs \his hands over \the [src] in appreciation of its contents.", \

```

Comment:
True, but since you're editing this line anyway you may as well remove it (I'm not asking you to remove every instance of it, that would be ridiculous XD)